cms/djangoapps/contentstore/views/item.py

```diff
@@ -333,6 +336,32 @@ def xblock_outline_handler(request, usage_key_string):
         return Http404
 
 
+# pylint: disable=unused-argument
```

Comment:
Putting the pylint disable pragma at this indentation is going to disable it for the rest of the file, or until it's re-enabled. Can you scope this down a bit, so that we're only disabling the lines that we actually intend to disable?